# Disable symbol order verification on macOS since building libc++.a adds extra symbols for some reason

--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -1024,7 +1024,7 @@ if (is_win) {
     }
   }
 
-  if (!is_asan && !is_component_build) {
+  if (false) {
     action("verify_chrome_framework_order") {
       script = "//chrome/tools/build/mac/run_verify_order.py"
       stamp_file = "$target_out_dir/run_$target_name.stamp"
